middleware/tokenauth/data/ec256-private.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
I'm assuming all the files in the `data` folder are used for testing, correct? If so, perhaps we should rename that folder to something like `test_certs` or something so that people don't get confused and want to use them in production, which would be bad.

Reply:
sure will do.